cloud-common/build.gradle

```diff
@@ -66,10 +66,29 @@ dependencies {
     compile group: 'net.sf.json-lib',                       name: 'json-lib',                       version: '2.4',  classifier: 'jdk15'
 
     testCompile group: 'junit',                             name: 'junit',                          version: junitVersion
+    testCompile group: 'org.junit.jupiter',                 name: 'junit-jupiter-api'
+    testCompile group: 'org.junit.jupiter',                 name: 'junit-jupiter-params',           version: junitJupiterVersion
     testCompile group: 'org.mockito',                       name: 'mockito-core',                   version: mockitoVersion
+    testImplementation group: 'org.mockito',                name: 'mockito-junit-jupiter'
     testCompile group: 'org.springframework.boot',          name: 'spring-boot-starter-test',       version: springBootVersion
     testCompile group: 'org.awaitility',                    name: 'awaitility',                     version: '3.1.6'
 
+    testRuntime group: 'org.junit.jupiter',                 name: 'junit-jupiter-engine'
+    testRuntime group: 'org.junit.vintage',                 name: 'junit-vintage-engine'
+
+    constraints {
```

Comment:
why?
I think version selectors are in a good place in the `dependency` closure, the `constraints` closure is just redundant

Reply:
Not sure, copied that part from elsewhere. Will remove it then.